### ISSUE TITLE
[enhancements] The whole process and UI needs to be revamped for simplicity

### ISSUE THREAD

**Issue by Mozoloa** [CiaraStrawberry/TemporalKit#31]
First of all thanks a lot for your work, I haven't yet managed to make it work but it seems promising.

I have a lot to say about the UI tho, this is my opinion but as a UX builder, the whole process is very confusing (even after 3 videos tutorials) and you could probably do away with keeping most of the settings hidden away and set automatically depending on the video. The less the users can fiddle around, the less errors there will be and the easier it will be to use, update and maintain.

Also we usually say that you should make your UI based on what the end user will want to do in the most basic natural langage possible, not directly giving full control of every parameter, especially using technical terms. You should only have to watch tutorials on how to finetune the settings, not make the whole thing work. A good rule of thumbs is if you can't explain it clearly under a 15 words hint and without requiring math skills then it should be auto set (based on the input video or some preset dropdown) and tucked away in some advanced settings

Some details:
- Globally, add some detailed hints on mouse hover
- Height and fps should be automatically defined based on the video itself and other settings
- You should be able to split the entire video into X amount of keyframes and it would be divided equally between first frame and last frame, with some rounding if need be
- Sides should be a simple frames per grid slider that would show an updated number of frames per grid and how many grids, based on the above
- frames per keyframe, I still don't know what it does exactly even after watching your videos, and it's not in the readme
- Target folder should show only if ebsynth or batch is selected
- Max keyframes and Border keyframes, not sure what it does, very confusing and can probably be hidden and auto set, or set another way
- I understand split based on cuts but people probably won't (hint would help)
- What does interpolate do ?
- EBSynth process has "batch size" and "max frame" if you don't have "read last settings" working, who knows what you're supposed to do with that
- Why do we need two buttons for prepare and recombine ? What do they do exactly ? (btw this doesn't work here, it throws `IndexError: list index out of range` altho my keys folder is filled with images)

Again this is my opinion, so it's not worth much, but I've been running those tools for a while and made some UXs around them, I'm not too dumb yet 75% of the Temporal Kit UX makes me think I'm stupid lmao

**Comment by uisato** [comment]
I agree. Spent most of last night trying to make it work and couldn't. I saw every video available on it and still couldn't. Looking forward some changes in the UI!

**Comment by CiaraStrawberry** [comment]
You're completely right, feature creeped myself ahah, same problem with the code itself, needs a big refactor, am planning that after some other stuff im doing right now, i appreciate the well formatted list :D

**Comment by merodiro** [comment]
> EBSynth process has "batch size" and "max frame" if you don't have "read last settings" working, who knows what you're supposed to do with that

I had "read last setting" failing on me a few times and I had no idea what value to set for these
